clang/test/CIR/CodeGen/OpenCL/spirv-target.cl

```diff
@@ -0,0 +1,29 @@
+// See also: clang/test/CodeGenOpenCL/spirv_target.cl
+// RUN: %clang_cc1 -cl-std=CL3.0 -fclangir -emit-cir -triple spirv64-unknown-unknown %s -o %t_64.cir
+// RUN: FileCheck --input-file=%t_64.cir %s --check-prefix=CIR-SPIRV64
+// RUN: %clang_cc1 -cl-std=CL3.0 -fclangir -S -emit-llvm -triple spirv64-unknown-unknown %s -o %t_64.ll
```

Comment:
Don't know which `cl-std` should we use. I think 3.0 is okay as we always have back compatibility, right?

Reply:
👍 3.0 makes sense.